frontend/public/locales/zh/machine-sets.json

```diff
@@ -0,0 +1,27 @@
+{
+  "Edit Machine count": "编辑机器计数",
+  "{{resourceKind}} maintain the proper number of healthy machines.": "{{resourceKind}} 保持正常数量的健康机器。",
+  "Create MachineAutoscaler": "创建 MachineAutoscaler",
+  "machine_0": "机器",
+  "Desired count": "所需数量",
+  "Current count": "当前数量",
+  "The most recently observed number of replicas.": "最近观察到的副本数。",
+  "Ready count": "就绪计数",
+  "The number of ready replicas for this MachineSet. A machine is considered ready when the node has been created and is ready.": "此 MachineSet 的可用副本数。当节点被创建并就绪时，机器被视为已就绪。",
+  "Available count": "可用数",
+  "The number of available replicas (ready for at least minReadySeconds) for this MachineSet.": "此 MachineSet 可用的副本数（至少适用于 minReadySeconds）。",
+  "Machine set details": "机器集合详情",
+  "Selector": "选择器",
+  "Machine role": "机器集",
+  "Region": "区域",
+  "Availability zone": "可用区",
+  "Name": "名称",
+  "Namespace": "命名空间",
+  "Machines": "Machine",
```

Comment:
If "Machine" needs to be translated into Chinese? It's translated in ja/machine-sets.json.

Reply:
This has not been updated.

Reply:
I can see that this had been addressed in the translation files, will send the files over again just in case.

Reply:
Looks good to me! Thanks.